server/game/cards/events/03/evenhandedjustice.js

```diff
@@ -0,0 +1,65 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../../drawcard.js');
+
+class EvenHandedJustice extends DrawCard {
+
+    canPlay(player, card) {
+        if(player !== this.controller || this !== card) {
+            return false;
+        }
+
+        return (_.every(this.game.getPlayers(), this.hasStandingCharacters)
+                && super.canPlay());
+    }
+
+    hasStandingCharacters(player) {
+        return (player.cardsInPlay
+                .filter(card => card.getType() === 'character' && !card.kneeled)
+                .length > 0);
+    }
+
+    play(player) {
+        var waitingPrompt = 'Waiting for opponent to use ' + this.name;
+
+        this.toKneel = [];  // characters to kneel (later)
+
+        // kneel opponent character
+        this.game.promptForSelect(player, {
+            activePromptTitle: 'Select a standing character controlled by your opponent',
+            waitingPromptTitle: waitingPrompt,
+            cardCondition: card =>
+                !card.kneeled
+                && card.getType() === 'character'
+                && card.controller !== this.controller,
+            onSelect: (player, card) => this.toKneel.push(card),
+        });
+
+        // kneel controller character
+        this.game.promptForSelect(player, {
+            activePromptTitle: 'Select a standing character of yours',
+            waitingPromptTitle: waitingPrompt,
+            cardCondition: card =>
+                !card.kneeled
+                && card.getType() === 'character'
+                && card.controller === this.controller,
+            onSelect: (player, card) => this.onCardsSelected(player, card),
+        });
+    }
+
+    onCardsSelected(player, card) {
+        this.toKneel.push(card);
+        _.each(this.toKneel, card => card.controller.kneelCard(card));
```

Comment:
You should probably return without kneeling anything here unless the `toKneel` array contains cards for both players. Based on the way you've structured the rest of the card it might not be possible for that to happen, but if the card is restructured it might be (e.g. click 'Done' without selecting a card when I select my card, but select one for the opponent, thus kneeling only their card).